pkg/tso/tso.go

```diff
@@ -48,6 +48,10 @@ const (
 	// and trigger unnecessary warnings about clock offset.
 	// It's an empirical value.
 	jetLagWarningThreshold = 150 * time.Millisecond
+
+	// minStorageTimeout is the minimum timeout for saving timestamp to storage.
+	// it keeps with the defaultTSOSaveInterval - 1 second to ensure the leader has enough time to allocate more tso after save successfully.
```

Comment:
If it is related to tso save interval, shall we use tso save interval to calculate this value?